app/src/main/java/com/fastlink/zekrapp/navigation/Navigation.kt

```diff
@@ -0,0 +1,32 @@
+package com.fastlink.zekrapp.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavController
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.fastlink.zekrapp.ui.favoriteScreen.FavoriteScreen
+import com.fastlink.zekrapp.ui.homeScreen.HomeScreen
+import com.fastlink.zekrapp.ui.zekirScreen.ZekirScreen
+
+@Composable
+fun Navigation(
+    navController: NavController,
+) {
+    NavHost(
+        navController = navController as NavHostController,
```

Comment:
If you want it as NavHostController then just make the navController in the parameter as a NavHostController